joschahenningsen/TUM-Live
# Edit Video Sections

### Motivation and Context
Closes Issue: #742 

### Description
- Added put request to video section endpoint.
- Updated the video section admin UI.
- Removed the `VideoSectionClient` class since it is a burden to work with it.

If the timestamp of a video section is changed a new thumbnail will not be generated.

### Steps for Testing
1. Log in as admin
2. Course Edit > Edit Course
3. Update video section name and timestamp
4. Head to the watch page and check if everything works as expected. 

### Screenshots
<div style="display: flex">
<img width="400" alt="Screenshot 2022-10-31 at 07 52 29" src="https://user-images.githubusercontent.com/29633518/198948806-18f1f4e6-c985-4614-856e-c21adefa1cc6.png">

<img width="400" alt="Screenshot 2022-10-31 at 07 52 19" src="https://user-images.githubusercontent.com/29633518/198948824-a987f5b1-5643-417f-ae7b-e0a9b2cb66bb.png">
</div>
